Calendar/BookingHelpers/BookingTenantSettingsHelper.ps1

```diff
@@ -0,0 +1,106 @@
+﻿function Get-BookingTenantSettings {
+    param([string] $domain)
+
+    if ($Script:MSSupport) {
+        $script:OrgConfig = Get-OrganizationConfig -Organization $domain
+        $script:OWAMBPolicy = Get-OwaMailboxPolicy -Organization $domain
+        $script:AcceptedDomains = Get-AcceptedDomain -Organization $domain
+    } else {
+        $script:OrgConfig = Get-OrganizationConfig
+        $script:OWAMBPolicy = Get-OwaMailboxPolicy
+        $script:AcceptedDomains = Get-AcceptedDomain
+    }
+    $ewsSettings = Get-EWSSettings $script:OrgConfig
+    $bookingsSettings = Get-BookingsSettings $script:OrgConfig
+    $OWAMBPolicy = Get-OwaMBPolicy $script:OWAMBPolicy
+    $acceptedDomains = Get-AcceptedDomains $script:AcceptedDomains
+    # Define the structure of the tenant settings
+    $tenantSettings = [PSCustomObject]@{
+        Identity         = $org.Identity
+        Guid             = $org.Guid
+        DisplayName      = $org.DisplayName
+        IsDeHydrated     = $org.IsDeHydrated
+        EWSSettings      = $ewsSettings
+        BookingsSettings = $bookingsSettings
+        OWAMBPolicy      = $OWAMBPolicy
+        AcceptedDomains  = $acceptedDomains
+    }
+
+    # Return the tenant settings
+    return $tenantSettings
+}
+
+function Get-EWSSettings {
+    param($org)
+    # Define the structure of the EWS settings
+    $EwsSettings = [PSCustomObject]@{
+        EwsAllowList               =$org.EwsAllowList
+        EwsApplicationAccessPolicy =$org.EwsApplicationAccessPolicy
+        EwsBlockList               =$org.EwsBlockList
+        EwsEnabled                 =$org.EwsEnabled
+    }
+
+    # Return the EWS settings
+    return $EwsSettings
+}
+
+function Get-BookingsSettings {
+    param($org)
```

Comment:
>$org [](http://example.com/codeflow?start=10&length=4)

I would upgrade this to OrgConfig or something like this so that it is clear it is more than the org name. (everywhere)